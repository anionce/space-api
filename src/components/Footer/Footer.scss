.footer {
	display: flex;
	justify-content: center;
	margin: 36px;

	img {
		height: 200px;
	}
}
