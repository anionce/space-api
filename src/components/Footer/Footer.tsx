import React from 'react';
import footer from '../../assets/img/footer.svg';
import './Footer.scss';

export const Footer = () => {
	return (
		<div className='footer'>
			<img alt='footer-logo' src={footer}></img>
		</div>
	);
};
