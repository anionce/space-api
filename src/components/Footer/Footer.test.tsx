import React from 'react';
import { screen } from '@testing-library/react';

import { customRender } from '../../assets/tests/utils';
import { Footer } from './Footer';

test('renders correcly', () => {
	const { container } = customRender(<Footer />);

	expect(container).toMatchSnapshot();
});

test('renders elements correctly', () => {
	customRender(<Footer />);

	expect(screen.getByAltText(/footer-logo/i)).toBeInTheDocument();
});
