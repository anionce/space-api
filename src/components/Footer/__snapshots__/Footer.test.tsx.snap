// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correcly 1`] = `
<div>
  <div
    class="footer"
  >
    <img
      alt="footer-logo"
      src="footer.svg"
    />
  </div>
</div>
`;
